modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/pages/additional_issues_table.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module PdfConstruction
+    module HigherLevelReview
+      module Pages
+        class AdditionalIssuesTable
+          MAX_ISSUES_ON_FIRST_PAGE = 13
```

Comment:
Is the only difference between the already existing AdditionalIssues class and this the MAX_ISSUES_ON_FIRST_PAGE?

Maybe we can move this to Structure::MAX_ISSUES_ON_FIRST_PAGE and have one file?

Reply:
It is for now, but there's a TODO later in the file to change this to output a table instead of just text output, so I felt like having it be its own file made sense.